src/partials/toolbar.hbs

```diff
@@ -6,6 +6,6 @@
 {{> breadcrumbs}}
 {{> page-versions}}
   {{#if page.editUrl}}
-  <div class="edit-this-page"><a href="{{page.editUrl}}">Edit this Page</a></div>
+  {{!-- <div class="edit-this-page"><a href="{{page.editUrl}}">Edit this Page</a></div> --}}
```

Comment:
If we're removing this, why not remove it completely? Or did I forget something that we discussed, earlier today?

Reply:
Why do you want to remove it at all?

Reply:
@mmattel, can you add your reasoning here, pls?

Reply:
pls see below